### PR TITLE
adding keyboard shortcuts

### LINKED ISSUE #55: Adding MeshMap keyboard shortcuts in the MeshMap docs.
### Current Behavior
Currently, we don't have MeshMap keyboard shortcuts in the MeshMap docs.

### Desired Behavior
We have to add the MeshMap keyboard shortcuts in the MeshMap docs.

### location in meshmap of keyboard shortcuts.
Screenshot 2023-11-11 at 12 21 04 AM

Screenshot 2023-11-11 at 12 20 13 AM

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

### PR DESCRIPTION
**Pull request by vaibhavpnimkar**
**Notes for Reviewers**

This PR fixes #55 
This PR for adding keyboard shortcuts in meshmap documention.

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by abhijeetgauravm** [comment]
Hey @vaibhavpnimkar Let's discuss this on website call tomorrow at 6:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by vaibhavpnimkar** [comment]
Added!!

**Comment by vaibhavpnimkar** [comment]
okay thanks for the feedback @leecalcote i will add Key Visualizations tomrrow

**Comment by vaibhavpnimkar** [comment]
i have  implemented your suggestions  plz have a look @leecalcote 
thankyou

**Comment by leecalcote** [comment]
Super. Thanks much for this, @vaibhavpnimkar 👍

File: content/en/meshmap/reference/keyboard-shortcuts.md [hunk 1]
```diff
@@ -1,21 +1,56 @@
...
+  Learn the keyboard shortcuts for Meshmap to enhance your designing experience.
+date: 2023-11-12
 ---
 
+Meshmap provides several keyboard shortcuts to improve your design workflow. These shortcuts can help you quickly perform various actions and tasks. Here is a list of commonly used keyboard shortcuts:
 
-You can copy and paste one or more components — even if you're going from one design to another.
+### Designer Shortcuts
+
+As a designer in Meshmap, you can take advantage of various keyboard shortcuts to streamline your design process. These shortcuts are designed to help you perform common actions quickly and efficiently:
```
Comment (leecalcote) [inline content/en/meshmap/reference/keyboard-shortcuts.md:12]: Nice write-up here. We seem to have lost the content menu description, though (the right-click menu). Will you add it back in? You can enhance it's description, too, if you like and include screenshot, even.

File: content/en/meshmap/reference/keyboard-shortcuts.md [hunk 2]
```diff
@@ -1,21 +1,56 @@
...
+
+- <kbd style="background-color: #6C757D; color: white;">Ctrl</kbd> + <kbd style="background-color: #6C757D; color: white;">M</kbd>: Add a comment to the selected node.
```
Comment (leecalcote) [inline content/en/meshmap/reference/keyboard-shortcuts.md:14]: Let's avoid repeating ourselves and define a new CSS class in _styles_project.scss, if you would.

**Comment by leecalcote** [comment]
I like your use of `kbd`. Pretty cool, @vaibhavpnimkar

**Pull request merged**